From: Helmut Grohne <helmut@subdivi.de>
Date: Tue, 7 Oct 2014 23:07:19 +0200
Subject: build-sys: check for libtoolize rather than libtool

The system libtool program has architecture dependent behaviour. It is
therefore unavailable in cross build environments. The only place it was
used in util-linux is autogen.sh to determine the availability of
libtool. All other places correctly use libtoolize or
$(top_builddir)/libtool.

Signed-off-by: Helmut Grohne <helmut@subdivi.de>
---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index d1a686c..d0b340f 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -66,7 +66,7 @@ test -f sys-utils/mount.c || {
 	echo
 	DIE=1
 }
-(libtool --version) < /dev/null > /dev/null 2>&1 || {
+(libtoolize --version) < /dev/null > /dev/null 2>&1 || {
 	echo
 	echo "You must have libtool-2 installed to generate util-linux build system."
 	echo
